src/main/java/net/bootsfaces/component/inputText/InputTextRenderer.java

```diff
@@ -260,25 +258,23 @@ public void encodeEnd(FacesContext context, UIComponent component) throws IOExce
 			numberOfDivs--;
 		}
 
-		// The following lines fix issue #1079 on basic tags (without typeahead).
-		// They initialize tagsinput manually and empty duplicated 'name' attribute
-		// on generated input (which holds the original HTML id).
 		if (inputText.isTags() && (!inputText.isTypeahead())) {
 			String id = fieldId; // input id
 			id = id.replace(":", "\\\\:"); // escape the id for jQuery
 			rw.startElement("script", null);
-			String js = "$('#" + id + "').tagsinput();" + //
-			            "$('#" + id + "').attr('name','');";
+			String js = "$('#" + id + "').tagsinput();";
+			js +=  "$('#" + id + "').siblings('.bootstrap-tagsinput').addClass('form-control');";
```

Comment:
with the update to 0.8.0 the bug with the 'name' was fixed, but there is a new bug fix a missing class